### PR TITLE
security validations FAQ

### PR DESCRIPTION
**Pull request by rosemarymarano**
As per     OWLS-91424 - Document - Security validation in WebLogic console, created a FAQ to address the new security validations/warnings in WLS.

### CODE REVIEW

File: documentation/staging/content/faq/security-validation.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+- For Domain in PV, use the WebLogic Scripting Tool (WLST), WebLogic Server Administration Console, or WebLogic Deploy Tooling (WDT) to make the recommended configuration changes.
+
+- For Domain in Image, create a new image with the recommended configuration changes.  
+
+- For Model in Image, make configuration changes in the model.
+
+     - For Model in Image domains, you may want to use [runtime updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}).
+
+     - For other domain home source types, you may want to use [configuration overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}}).
```
Comment (tbarnes-us) [inline documentation/staging/content/faq/security-validation.md:30]: How about:

You may need to make the recommended configuration changes using an approach that depends on your [domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}):

- For Domain in PV, use the WebLogic Scripting Tool (WLST), WebLogic Server Administration Console, WebLogic Deploy Tooling (WDT), or [configuration overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}}).

- For Domain in Image, create a new image with the recommended changes or use [configuration overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}}).

- For Model in Image, supply model files with the recommended changes in its image's `modelHome` directory or use [runtime updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}).

Comment (rosemarymarano) [reply documentation/staging/content/faq/security-validation.md]: YES! definitely pulls together the information much more coherently. Thank you. See https://github.com/oracle/weblogic-kubernetes-operator/pull/2495/commits/156df2b67fb9f64ade5e580b185fcc70e9f4a9e8

File: documentation/staging/content/faq/security-validation.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+>
+> SOLUTION: Update the user lockout settings (LockoutThreshold, LockoutDuration) to be secure.
+
+WebLogic Server has a new, important feature to ensure and help you secure your WLS domains when running in production. With the July 2021 PSU applied, WebLogic Server regularly validates your domain configuration settings against a set of security configuration guidelines to determine whether the domain meets key security guidelines recommended by Oracle. For more information and additional details, see [MOS Doc 2788605.1](https://support.oracle.com/rs?type=doc&id=2788605.1) "WebLogic Server Security Warnings Displayed Through the Admin Console" and [Review Potential Security Issues](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/lockd/secure.html#GUID-4148D1BE-2D54-4DA5-8E94-A35D48DCEF1D) in _Securing a Production Environment for Oracle WebLogic Server_.
+
+Warnings may be at the level of the JDK, or that SSL is not enabled. They may recommend that you take an external action like setting the right permissions in the `DOMAIN_HOME` file system. Some warnings may recommend updating your WebLogic configuration.
```
Comment (mriccell) [inline documentation/staging/content/faq/security-validation.md:17]: I have opened https://jira.oraclecorp.com/jira/browse/OWLS-91448 addressing file permissions of the Domain Home.  We have found that in OpenShift when WDT creates the the Domain Home the Operator needs write permissions.  Not sure if this is only a OpenShift issue or also in other K8s vendors.

Comment (mriccell) [reply documentation/staging/content/faq/security-validation.md:17]: I would remove for now the sentence "They may recommend that you take an external action like setting the right permissions in the `DOMAIN_HOME` file system." until we haver more clarity

**Review by mriccell — CHANGES_REQUESTED** [review]

File: documentation/staging/content/faq/security-validation.md (hunk 2 above)
Comment (rosemarymarano) [reply documentation/staging/content/faq/security-validation.md]: Done. See https://github.com/oracle/weblogic-kubernetes-operator/pull/2495/commits/821a6e7bc2f1ded427f61db523bd39951d7c03d5

**Review by mriccell — APPROVED** [review]

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**